# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/obs-studio/hotfix-gcc15.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

obs-studio/deps/json11/json11.cpp:330:56: error: 'uint8_t' d oes not name a type

--- obs-studio/deps/json11/json11.cpp.vanilla	2025-05-13 21:15:37.383888886 +0200
+++ obs-studio/deps/json11/json11.cpp	2025-05-13 21:15:47.845477919 +0200
@@ -25,6 +25,7 @@
 #include <cstdlib>
 #include <cstdio>
 #include <limits>
+#include <inttypes.h>
 
 namespace json11 {
 
